README.md

```diff
@@ -44,6 +44,19 @@ without having to build it from source.
 TODO(https://github.com/mc-imperial/dredd/issues/28): add instructions
 on how to use Dredd
 
+`pi.cc` is a small example in `examples/simple` that calculates an approximation of pi. To run this example, run the following from the root of the repository:
+```
+# This will save the example file since pi.cc is modified in-place by dredd.
+cp examples/simple/pi.cc examples/simple/backup.cc
+third_party/clang+llvm-13.0.1/bin/dredd examples/simple/pi.cc
+clang examples/simple/pi.cc -lstdc++ -o examples/simple/pi
+```
+Running the executable with no additional arguments via `examples/simple/pi` will produce the expected result of 3.14159.  You can specify which mutants you want to use at runtime by running:
+```
+DREDD_ENABLED_MUTATION=$i$ examples/simple/pi
+```
+where $i$ is the id of the mutation in the program generated by dredd. This will print a different output to the original program and in some cases may not terminate.
```

Comment:
Before "This will print" I would add:

For example, running ... will enable mutation number 3.

so that they have a very concrete example.

Also: "This will print a different output" - change it to "This may", and remark that some mutants might be *equivalent* mutants, in which case the value printed to the console will be unchanged.